package.json

```diff
@@ -23,6 +23,7 @@
     "@toi/toi": "1.3.0",
     "@typeofweb/schema": "0.7.3",
     "@types/benchmark": "2.1.1",
+    "ajv": "^6.12.6",
```

Comment:
Please use a fixed version.

Renovate will keep them up-to-date.

Reply:
I didn't even notice that, done.